src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/JSON.txt

```diff
@@ -133,3 +133,75 @@ Error({Kind:ErrorKind.Div0})
 // Flattening nested tables
 >> JSON([[1,2,3],[4,5],[6]], JSONFormat.FlattenValueTables)
 "[[1,2,3],[4,5],[6]]"
+
+>> JSON(ParseJSON("{}"))
+"{}"
+
+>> JSON(ParseJSON("[]"))
+"[]"
+
+>> JSON(ParseJSON("1"))
+"1"
+
+>> JSON(ParseJSON("1.77"))
+"1.77"
+
+>> JSON(ParseJSON("-871"))
+"-871"
+
+>> JSON(ParseJSON("""John"""))
+"""John"""
+
+>> JSON(ParseJSON("true"))
+"true"
+
+>> JSON(ParseJSON("false"))
+"false"
+
+>> JSON(ParseJSON("{""a"": 1}"))
+"{""a"":1}"
+
+>> JSON(ParseJSON("{""a"": ""x""}"))
+"{""a"":""x""}"
+
+>> JSON(ParseJSON("[1]"))
+"[1]"
+
+>> JSON(ParseJSON("{""a"": 1.5}"))
+"{""a"":1.5}"
+
+>> JSON(ParseJSON("[1.5]"))
+"[1.5]"
+
+>> JSON(ParseJSON("{""a"":[1]}"))
+"{""a"":[1]}"
+
+>> JSON(ParseJSON("[{""a"": -17}]"))
+"[{""a"":-17}]"
+
+>> JSON(ParseJSON("[true, false]"))
+"[true,false]"
+
+>> JSON(ParseJSON("[""True"", ""False""]"))
+"[""True"",""False""]"
+
+>> JSON(ParseJSON("   {  ""a"" :  1 }  "))
+"{""a"":1}"
+
+>> JSON(ParseJSON("{""a"": {""a"": 1}}"))
+"{""a"":{""a"":1}}"
+
+// Depth 41
```

Comment:
Since we lowered the maxdepth to 20, the test should have only a depth of 21 (same for test on line 202)